### PR TITLE
Add severity levels based on Brakeman confidence

### PR DESCRIPTION
**Pull request by aergonaut**
I'm adding severity levels to the Pronto messages based on the confidence of the Brakeman warning.

Brakeman has 3 levels of confidence: high, medium and low. I'm mapping these to fatal, warning and info on the Pronto messages, respectively.

Hopefully bubbling up the confidence rating from Brakeman will help make the report from Pronto even more useful! 😃

### CODE REVIEW

**Comment by aergonaut** [comment]
Ping @mmozuras 😃

File: README.md [hunk 1]
```diff
@@ -6,3 +6,9 @@
 [![Dependency Status](https://gemnasium.com/mmozuras/pronto-brakeman.png)](https://gemnasium.com/mmozuras/pronto-brakeman)
 
 Pronto runner for [Brakeman](https://github.com/presidentbeef/brakeman), security vulnerability scanner for RoR. [What is Pronto?](https://github.com/mmozuras/pronto)
+
+## Severity mapping
+
+Brakeman [Confiedence](https://github.com/presidentbeef/brakeman#confidence-levels) is mapped to severity levels on the
```
Comment (mmozuras) [inline README.md]: Confiedence -> Confidence 😄

**Comment by mmozuras** [comment]
@aergonaut sorry about that, a busy month 😄 

What about: high -> fatal, medium -> error, low -> warning?

**Comment by aergonaut** [comment]
@mmozuras my problem with that mapping is that the `GithubStatusFormatter`'s [mapping](https://github.com/mmozuras/pronto/blob/master/lib/pronto/formatter/github_status_formatter/status_builder.rb#L63-L68) turns warnings into failed status marks. So a low confidence Brakeman warning would generate a red mark on a PR; to me low confidence seems more like something you should look at, but not something that is very urgent.

How about if I move medium confidence warnings up to the error level, but leave low confidence at the info level? There's still a skipped level, but it's lower on the chain.

**Comment by mmozuras** [comment]
@aergonaut you explanation makes sense. Let's leave it as you've first suggested.

Thanks for the contribution! 🙇

**Comment by aergonaut** [comment]
Thanks 🎉

**Pull request merged**